minio/Makefile

```diff
@@ -0,0 +1,8 @@
+stop:
+	docker-compose down -v
+
+file-ops:
+	docker-compose up file-ops
```

Comment:
the `command` doesn't stay up, but exists, so this should be a `docker-compose run` command

Reply:
done.